### DISCUSSION TITLE
Dimmable lights

### DISCUSSION THREAD

**Discussion by OrigSorceror** [autoSteve/acMqtt#55] [General]
not really an issue as COVER does the job just looks wierd in HAOS. 

I have quite a few Dimmable LED light connected to Dim contollers. 

Discovered the Cover keyword will allow control of dimmable lights in HAOS with a slider similar to what the SHAC displays on the 5000ETP10W panel. (wich i am getting rid of hence the reason for getting things up and running in HAOS) already got a 10" screen with an attached rPi3b running in Kiosk mode to replace the 5000ETP10W (in reality a XDS1078 room sign panel from Iadea running android 4.4 :( in a kiosk mode with a simple kiosk app by clipsal) hence why i want to dump it  as this version of Android is not secure. Also Hardware limits and continual crashes and limited customisation options (being restricted to SHAC).

Is it possible to create a dimmable light option with On Off as well as level slider. Using the cover keyword gives the slider control but produses the openstop close level control as well as the cover icon lol.

**Comment by OrigSorceror** [comment]
![image](https://github.com/user-attachments/assets/7196d48b-3db3-4e3d-b199-21fcb729e170)
![image](https://github.com/user-attachments/assets/29458b40-96da-41f6-8076-26f2a524c8f1)

**Comment by ziogref** [comment]
It's already possible to have sliding lights in Home Assistant.

Is this what you are after?
https://github.com/user-attachments/assets/15a34d6c-fe3b-442c-9b34-10a4d470f038

**Comment by OrigSorceror** [comment]
> It's already possible to have sliding lights in Home Assistant.
> 
> Is this what you are after? https://github.com/user-attachments/assets/15a34d6c-fe3b-442c-9b34-10a4d470f038

yeah something light that.. I am controlling them from the HA MQTT broker at the moment will play a little more

**Comment by ziogref** [comment]
So this is my setup in cbus for one of my dimmable lights

MQTT, light, sa=Living Room, pn=Bank 1, img=mdi:lightbulb, 

Does that help?

**Reply by OrigSorceror** [reply]
yes that is similar to how i have it set up.  Controlling it from the Mosquitto broker does not offer the slid bar. but controlling it from the overview page or seperate card veiw page (need to set these up)  does offer the slider. and can set up cards with the slifer already visable.. just a newb at HA,  as i use HomeBridge for everything.. only need HA to replace the 5500SHAC clunky web layout and config and ditch the 5000ETP10W. in favour of a 10 pi driven display in Kiosk mode

Main reason is i found out that HAOS has themes.  And they look so much better than my Trek style theme i currently use in SHAC. and a darn site easier to configure and arange.

my current SHAC Page on the 5000ETP10W
![image](https://github.com/user-attachments/assets/2c5693eb-344a-453b-a079-093eedd29227)

about once a week or so it ends up resetting back to the default  
![image](https://github.com/user-attachments/assets/c1d7c393-1598-47c0-a635-abda39d50a71)

hence why i want to do away with the clunky underpowered display and the clunky web config of SHAC

**Comment by autoSteve** [comment]
Continue here.
